From: Tony Jones <tonyj@suse.de>
Date: Mon 01 Jun 2020 11:35:26 PM CEST
Subject: resolve KABI warning for perf-pt-coresight
References: git-fixes
Patch-mainline: never, kabi
Signed-off-by: Tony Jones <tonyj@suse.de>

Fix KABI for patches.suse/perf-pt-coresight-fix-address-filters-for-vmas-with-non-zero-offset.patch

--- a/include/linux/perf_event.h.orig
+++ b/include/linux/perf_event.h
@@ -671,7 +671,11 @@
 	/* address range filters */
 	struct perf_addr_filters_head	addr_filters;
 	/* vma address array for file-based filders */
+#ifdef __GENKSYMS__
+	unsigned long			*addr_filters_offs;
+#else
 	struct perf_addr_filter_range	*addr_filter_ranges;
+#endif
 	unsigned long			addr_filters_gen;
 
 	void (*destroy)(struct perf_event *);
